docs/config/conf.py

```diff
@@ -22,8 +24,12 @@
 author = "Intel"
 
 # The full version, including alpha/beta/rc tags
-release = "0.11.0"
-
+release = os.getenv("UMF_VERSION", "")
+print(
+    f"UMF_VERSION: {release}"
```

Comment:
to be precise I'd write `UMF_VERSION used in docs: `

Reply:
done